### PR TITLE
[New UI] Update the new UI of dev document

### PR DESCRIPTION
**Pull request by QuakeWang**
Updated website with new UI screenshots

### CODE REVIEW

**Comment by QuakeWang** [comment]
@zhongjiajie PTAL

**Comment by zhongjiajie** [comment]
Thanks for your contribution, I will take a look tomorrow.

File: docs/en-us/dev/user_doc/guide/datasource/hive.md [hunk 1]
```diff
@@ -2,9 +2,7 @@
 
 ## Use HiveServer2
 
- <p align="center">
-    <img src="/img/hive-en.png" width="80%" />
-  </p>
+![hive](/img/new_ui/dev/datasource/hive.png)
```
Comment (zhongjiajie) [inline docs/en-us/dev/user_doc/guide/datasource/hive.md:5]: Good point to add new directory to keep new ui

**Review by zhongjiajie — APPROVED** [review]
LGTM, well done

File: docs/en-us/dev/user_doc/guide/datasource/hive.md (hunk 1 above)
Comment (zhongjiajie) [reply docs/en-us/dev/user_doc/guide/datasource/hive.md:5]: also good to use markdown syntax

Comment (QuakeWang) [reply docs/en-us/dev/user_doc/guide/datasource/hive.md:5]: > Good point to add new directory to keep new ui

It's good for the organization and standardization of the images later on, so I chose to create a new catalog.

**Pull request merged**